gprofiler/profilers/perf.py

```diff
@@ -567,7 +567,10 @@ def relevant_for_process(self, process: Process) -> bool:
         return is_golang_process(process)
 
     def make_application_metadata(self, process: Process) -> Dict[str, Any]:
-        metadata = {"golang_version": get_process_golang_version(process)}
+        metadata = {
+            "golang_version": get_process_golang_version(process),
+            "stripped": elf_is_stripped(f"/proc/{process.pid}/exe"),
```

Comment:
Hmm, I figured that this lacks `raise_nosuchprocess`, doesn't it? Please add in utils